### DISCUSSION TITLE
Cannot choose modified preset from HMI screen

### DISCUSSION THREAD

**Discussion by downtime358** [jilleb/MQB-FPA#20] [Modified profiles]
Hi,

first of all, super thanks for the excellent guide, template and script you've made.

I have an issue and I'm not figuring out why this happens. I used the template against the data and modified the FPA dataset so that Eco mode is replaced with Comfort mode and I also modified the contents of it that it actually change the ride comfort etc.
I have a retrofitted DCC and this modification seems to now work.

However, there's a slight issue with selecting the comfort mode. When using mode button I can scroll through all the modes and I can select the comfort through that. If I just press mode button once to bring up the menu and try to select the Comfort preset from the HMI display, it just flashes that it recognizes the push but it does not select the mode.
Am I missing something from the modification?

**Comment by downtime358** [comment]
Ok, this I got fixed.

I rewrote the original unchanged dataset and after that wrote the dataset with just the modified FPA, not the complete dataset.

Now it works perfectly 👯

**Reply by mziker** [reply]
bonjour pourriez vous partager votre fichier merci
